### PR TITLE
37 subscribe app

### PR DESCRIPTION
**Pull request by yuriiZherebukh**
## TRELLO

* [Main Trello ticket](https://trello.com/b/BJDdC443/mytrip)

## Code reviewers

- [ ] @Ashiato
- [ ] @black-art
- [ ] @hrytskivr
- [ ] @KarnaushenkoVO
- [ ] @PanLeon
- [ ] @yuriiZherebukh

### Second Level Review

- [ ] @byshovets
- [ ] @lhalam

## Summary of issue

Created Subscribe app with:
1. Models
2. Views
3.Urls
4.Tests

## Summary of change

Fixed route to Help app at global routes.

## Testing approach

All test passed, pylint 9.6/10.

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: myTrip/subscribe/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('trip', '0002_auto_20170720_1121'),
```
Comment (lhalam) [inline myTrip/subscribe/migrations/0001_initial.py]: '0002_auto_20170720_1121'=>0001_initial

File: myTrip/subscribe/views.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        Args:
+            request.user.id(int): id of logged user,
+            trip_id (int): id of trip,
+            subscribed_id (int): id of user, was subscribed by logged user.
+        Returns:
+            JsonResponse: response: <Subscribe>
+            or
+            HttpResponse: status: 404.
+        """
+        if subscribed_id:
```
Comment (lhalam) [inline myTrip/subscribe/views.py]: todo

File: myTrip/subscribe/views.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        Args:
+            request.user.id(int): id of logged user,
+            trip_id (int): id of trip,
+            subscribed_id (int): id of user, on who logged user wishes to subscribe.
+        Returns:
+            JsonResponse: response: <Subscribe>
+            or
+            HttpResponse: status: 404.
+        """
+        user = CustomUser.get_by_id(user_id=request.user.id)
```
Comment (lhalam) [inline myTrip/subscribe/views.py:64]: user = request.user

File: myTrip/subscribe/views.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        if user is None:
```
Comment (lhalam) [inline myTrip/subscribe/views.py]: delete this logic

**Review by lhalam — COMMENTED** [review]
ToDo

**Review by hrytskivr — APPROVED** [review]
approved

**Pull request merged**